### ISSUE TITLE
what is warm start image?

### ISSUE THREAD

**Issue by castleKing1997** [astro-informatics/purify#265]
there's a parameter named "warm_start".
it needs a path/to/warm/start/image
we cannot find any interpretation of this
so what will a warm start image be?

**Comment by Luke-Pratley** [comment]
@castleKing1997 Good point... I need to add documentation on this! I will add some. The warm start is the ability to start from a previous image run with of purify using the same settings. When you don't have a previous run, replace it with two quotations `""`.
